### PR TITLE
RUN-2008: runner name

### PR DESCRIPTION
**Pull request by smartinellibenedetti**
**Is this a bugfix, or an enhancement? Please describe.**
Fix for runner name (need a second PR on pro side);

Originally had flattened the settings to solve a console error, but this caused another issue that was preventing the runner name from showing. I've undone the change and saw that the console error isn't showing up anymore. 

Will make a second PR on pro side to ensure that the runner name will show up

### CODE REVIEW

File: rundeckapp/grails-spa/packages/ui-trellis/src/library/components/execution-log/logViewer.vue [hunk 1]
```diff
@@ -49,9 +49,11 @@
...
         </div>
-        <ui-socket section="execution-log-viewer" location="settings"
-                   :event-bus="eventBus"
-        />
+        <div class="vue-ui-socket">
+          <ui-socket section="execution-log-viewer" location="settings"
+                     :event-bus="eventBus"
+          />
+        </div>
       </form>
```
Comment (gschueler) [inline rundeckapp/grails-spa/packages/ui-trellis/src/library/components/execution-log/logViewer.vue:57]: why this change, did it affect things? i don't think we should wrap ui-socket like that when it is already inside a Vue component

Comment (smartinellibenedetti) [reply rundeckapp/grails-spa/packages/ui-trellis/src/library/components/execution-log/logViewer.vue:57]: Just updated it to match the format recommended for the vue 3 migration (ui-socket inside of another element with vue-ui-socket class) but it doesn't make any difference how things are rendered in practice

Comment (smartinellibenedetti) [reply rundeckapp/grails-spa/packages/ui-trellis/src/library/components/execution-log/logViewer.vue:57]: I can undo it if you'd like

Comment (gschueler) [reply rundeckapp/grails-spa/packages/ui-trellis/src/library/components/execution-log/logViewer.vue:57]: yes, that format is only needed when embedding ui-socket into the page outside of a Vue component

**Review by gschueler — APPROVED** [review]
